src/pg/db/config.js

```diff
@@ -0,0 +1,27 @@
+import debug from 'debug';
```

Comment:
(minor) To use the debug library, you must do this:

```js
import debug from 'debug';

const log = debug('graffy:pg:config'); // add a tag for all the lines logged from this file.

log(...); // This is the function you call to log things.
```

Later, when running your program, you will see no log lines. To enable them, you have to set an environment variable:

```sh
DEBUG=graffy:pg:*
```

You can also do `DEBUG=*` but it will be very noisy (Graffy is not the only library using `debug`).

Reply:
what is the convention here?
at this particular file I use 
`debug(`graffy:pg:error`)(
          `Error in Postgres ConnectionPool: ${err.message}`,
        );` 
isn't it equivalent with 
`const log = debug('graffy:pg:error');
log(`Error in Postgres ConnectionPool: ${err.message}`)`
?